### ISSUE TITLE
Aggregate statistics on ladder leaderboards don't include walkovers/forfeits

### ISSUE THREAD

**Issue by vkorablin** [Starbow/StarbowWebSite#7]
As far as I understand, this is done on purpose:

```
        ladder_wins='(stats.ladder_wins-stats.ladder_walkovers) as ladder_wins',
        ladder_losses='(stats.ladder_losses-stats.ladder_forefeits) as ladder_losses',
```
source: https://github.com/Starbow/StarbowWebSite/blob/90611021c2f0a50cf4fbfaf792729b3e9b3273f3/starbowmodweb/ladder/views.py#L44

This wasn't a big deal during the times of standalone eros client, but now it's common to forfeit instead or waiting for the winner to upload a replay. Should we fix this?

**Comment by Veritasimo** [comment]
Discouraging people from forfeiting is probably a better solution. Or, retroactively fixing forfeits to losses upon upload of a valid replay.

**Comment by vkorablin** [comment]
Well, talking about proper solutions, using a player's match history through Blizzard's API seems the best. One or both players press a button that says "match over", the server looks up the match in one of their match histories. No need for replay upload at all.

Now if only you could get a link to the player's in-game profile the same way... (Impossible so far as I've gathered.)

**Comment by Veritasimo** [comment]
I assumed they were going to switch to a Blizzard API lookup since we couldn't automate replays. That being said, I remember there were a few issues with that approach. I believe the API is cached on their end, so it doesn't update instantly. Also, the match details available are incredibly scarce on the API (or at least they were when I last looked into it).

Doubtless there are probably people more informed than me as to the group consensus on this.

**Comment by Sikian** [comment]
I think most of the issues we had with the API are no longer there due to the upgrades Blizzard have done on their side. The only issue I saw was that there was no uid for the matches, so we'd have to compare the timestamps of the games of both players, which would required a bit more logic. 

Regardless, I'd go for the API solution as soon as we can.

**Comment by vkorablin** [comment]
There's also no race info. Kinda sucks that we'll lose statistics, but that's a sacrifice that can be made IMO.

````
{
    "map" : "Starbow - Blue Storm 2.0",
    "type" : "CUSTOM",
    "decision" : "LOSS",
    "speed" : "FASTER",
    "date" : 1434289630
}
````

**Comment by Sikian** [comment]
Yeah, that sucks. I wonder why Blizzard doesn't provide that info. It's like they're publishing the bare minimum.
